test/apiv2/10-images.at

```diff
@@ -325,4 +325,21 @@ t DELETE images/$iid_test2?noprune=false 200
 t GET libpod/images/$iid_test1/exists 404
 t GET libpod/images/$iid_test2/exists 404
 
+# If the /resolve tests fail, make sure to use ../registries.conf for the
+# podman-service.
+
+# With an alias, we only get one item back.
+t GET libpod/images/podman-desktop-test123/resolve 200 \
+  .Names[0]="florent.fr/will/like:this"
```

Comment:
How can this work? Shouldn't this be `:latest`?

Reply:
Committed the wrong state -.-  I'm on it

Reply:
Should 🐝 better now

Reply:
Thanks. Tests still fail, so one suggestion for fixing, and one suggestion for better coverate:
```diff
diff --git a/test/apiv2/10-images.at b/test/apiv2/10-images.at
index a1b0fa170..2c1f734c9 100644
--- a/test/apiv2/10-images.at
+++ b/test/apiv2/10-images.at
@@ -340,6 +340,7 @@ t GET libpod/images/no-alias-for-sure/resolve 200 \
   .Names[2]="registry.fedoraproject.org/no-alias-for-sure:latest"
 
 # Test invalid input.
-t GET libpod/images/noCAPITALcharAllowed/resolve 400
+t GET libpod/images/noCAPITALcharAllowed/resolve 400 \
+  .cause="repository name must be lowercase"
 
 # vim: filetype=sh
diff --git a/test/apiv2/test-apiv2 b/test/apiv2/test-apiv2
index c1ec290df..c4d04d6d4 100755
--- a/test/apiv2/test-apiv2
+++ b/test/apiv2/test-apiv2
@@ -448,7 +448,8 @@ function start_service() {
         $PODMAN_BIN unshare true
     fi
 
-    $PODMAN_BIN \
+    CONTAINERS_REGISTRIES_CONF=$TESTS_DIR/../registries.conf \
+        $PODMAN_BIN \
         --root $WORKDIR/server_root --syslog=true \
         system service \
         --time 0 \
```

Reply:
I was convinced that all tests in CI are pointed to test/registries.conf.  Thanks!

Reply:
Ah.  OK, I recalled right.  The env var is set in CI where the test passes.  But I will add your patch as well so it'll just pass in dev environments as well.

Reply:
I was not aware that this was happening, but yeah. Actual override of `/etc` file, not an envariable: https://github.com/containers/podman/blob/0655bf3d34e105badcbc063a093668fd565fa1a4/contrib/cirrus/lib.sh#L200-L203